src/components/SideBar/SideBarBanner/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import Link from "next/link";
+import Image from "next/image";
+
+import styles from "./style.module.css";
+
+type Release = {
+  target: string;
+  image: string;
+  text: string;
+};
+
+type Props = {
+  release?: Release;
+};
+
+const SideBarBanner: React.FC<Props> = ({ release }) => {
+  if (!release) return <></>;
```

Comment:
`if (!release) return null;`

The empty fragment still like a div or something that React will have to render, null is nothing.